development/extensions/tutorial_basics.rst

```diff
@@ -74,341 +87,94 @@ look at the complete file:
+   ``version`` | "The version of your extension. This should follow the format of X.Y.Z with an optional suffix
+   of -dev, -patch, -alpha, -beta or -RC."
+   ``time`` *optional* | "The release date of your extension. Must be in YYYY-MM-DD or YYYY-MM-DD HH:MM:SS format."
+   ``keywords`` *optional* | "An array of keywords related to the extension."
+   ``license`` | "The license of the package. This can be either a string or an array of strings.
+   Typically extensions should be licensed under the same GPL-2.0 license as phpBB."
+   ``authors`` | "An array of authors of the extension.
    See `authors`_ for more details."
-   ``require`` | "An array with requirements of the extension.
+   ``require`` | "An array of requirements of the extension.
    See `require`_ for more details."
-   ``require-dev`` | "An array with development requirements of the extension.
+   ``require-dev`` *optional* | "An array of development requirements of the extension.
    See `require-dev`_ for more details."
-   ``extra`` | "An array with additional values of the extension.
+   ``extra`` | "An array of arbitrary extra data.
    See `extra`_ for more details."
 
 authors
 -------
 
-You may have unlimited authors. But you should at least have one.
+You may have unlimited authors. At least have one author is highly recommended.
 
 .. csv-table::
    :header: "Field", "Content"
    :delim: |
 
-   ``name`` | Name of the author
-   ``email`` | Email address of the author
-   ``homepage`` | Must contain one valid URL or be empty
-   ``role`` | "Role can be used to specify what the authors did for the
-   extension (e.g. Developer, Translator, Supporter, ...)"
+   ``name`` | "The name of an author."
+   ``email`` *optional* | "An email address of the author."
+   ``homepage`` *optional* | "A URL pointing to the website of the author."
+   ``role`` *optional* | "Role can be used to specify what the author did for the
+   extension (e.g. Developer, Translator, Supporter, etc.)"
 
 require
 -------
 
-In the ``require`` section you can specify the technical requirements of your
-extension. Examples are the ``php`` version, or
-`third party libraries <https://packagist.org/>`_. Since our demo extension does
-not require any additional library, we only use the PHP version requirement, to
-make sure people have the right PHP version on their server, and composer
-installers for some internal handling. phpBB 3.1 requires PHP 5.3.3 or higher,
-so the version comparison is ``>= 5.3.3``.
+List the dependencies required by the extension, i.e: the PHP version and
+`third party libraries <https://packagist.org/>`_.
+
+.. csv-table::
+   :header: "Field", "Content"
+   :delim: |
+
+   ``php`` | "The minimum-stability version of PHP required by the extension. phpBB 3.1 requires PHP 5.3.3 or higher,
+   so the version comparison is ``>= 5.3.3``."
+   ``composer/installers`` | "Recommended by phpBB for some internal handling."
```

Comment:
I agree, but then is telling that it is for some internal handling not the right term :)